Userland/Libraries/LibCrypto/Hash/SHA2.cpp

```diff
@@ -34,16 +47,16 @@ inline void SHA256::transform(u8 const* data)
         m[i] = (data[j] << 24) | (data[j + 1] << 16) | (data[j + 2] << 8) | data[j + 3];
     }
 
-    for (; i < BlockSize; ++i) {
+    for (; i < 64; ++i) {
```

Comment:
can we not use SHA256::BlockSize here? replacing a meaningfully named constant with a magic number is less than ideal.

Reply:
No, cannot read private SHAxxx::member constants form non-member function. The function needs to be non-member because the attribute target would not work otherwise. Will fix that by moving the constant from header into this function as constexpr variable.

Reply:
Fixed.